### PR TITLE
Adds git-add -p instructions

### PR DESCRIPTION
**Pull request by sethwoodworth**

### CODE REVIEW

File: pages/common/git-add.md [hunk 1]
```diff
@@ -17,3 +17,7 @@
 - Also add ignored files:
 
 `git add -f`
+
+- Add parts of a file interactively
+
+`git add -p {{PATHSPEC}}`
```
Comment (waldyrious) [inline pages/common/git-add.md]: Can we use something other than "PATHSPEC"? It doesn't have to be 100% technically correct, just cover the most common cases, while being intelligible to beginners.

Comment (ostera) [inline pages/common/git-add.md]: `path/to/file` should be enough.

Comment (igorshubovych) [inline pages/common/git-add.md]: Maybe better `{{path/in/repo}}` or `{{path/in/project}}`.

Comment (waldyrious) [inline pages/common/git-add.md]: I like @ostera's version more, I don't expect many people would try adding files from outside the project directory in order to justify @igorshubovych's caveat.

**Comment by sethwoodworth** [comment]
Agreed with everyone above.
Changed to `path/to/file` which definitely reads better, fixed a small lint error, squashed and updated.

**Comment by waldyrious** [comment]
Great :+1: LGTM!

**Comment by rubenvereecken** [comment]
Cool, thanks.

**Pull request merged**